qhub/schema.py

```diff
@@ -74,6 +74,13 @@ class HelmExtension(Base):
     overrides: typing.Optional[typing.Dict]
 
 
+# ============== Argo-Workflows =========
+
+
+class ArgoWorkflows(Base):
+    enabled: bool
```

Comment:
Would be nice like the other helm charts to support overrides here.

Reply:
added